docs/build/_index.md

```diff
@@ -1,9 +1,164 @@
+    <li id="c1_slide9">
+      <a href="/components/board/jetson/">
+        {{<imgproc src="get-started/installation/thumbnails/jetson-xavier.png" alt="Jetson Xavier NX Dev Kit" resize="148x120" declaredimensions=true >}}
+        <p>NVIDIA Jetson Xavier NX</p>
+      </a>
+    </li>
+    <li id="c1_slide10">
+      <a href="/get-started/installation/prepare/rpi-setup/">
+        {{<imgproc src="get-started/installation/thumbnails/rpi-3.png" alt="Raspberry Pi 3" resize="148x120" declaredimensions=true >}}
+        <p>Raspberry Pi 3</p>
+      </a>
+    </li>
+    <li id="c1_slide11">
+      <a href="/get-started/installation/prepare/rpi-setup/">
+        {{<imgproc src="get-started/installation/thumbnails/raspberry-pi-zero-2w.png" alt="Raspberry Pi" resize="148x120" declaredimensions=true >}}
+        <p>Raspberry Pi Zero 2 W</p>
+      </a>
+    </li>
+    <li id="c1_slide12">
+      <a href="/get-started/installation/prepare/beaglebone-setup/">
+        {{<imgproc src="get-started/installation/thumbnails/beaglebone.png" resize="148x120" declaredimensions=true alt="BeagleBone A I-64">}}
+        <p>BeagleBone AI-64</p>
+      </a>
+    </li>
+    <li id="c1_slide13">
+      <a href="/components/board/numato/">
+        {{<imgproc src="get-started/installation/thumbnails/numato.png" alt="Numato GPIO Modules" resize="148x120" declaredimensions=true >}}
+        <p>Numato GPIO Modules</p>
+      </a>
+    </li>
+    <li id="c1_slide14">
+      <a href="/components/board/pca9685/">
+        {{<imgproc src="get-started/installation/thumbnails/pca9685.png" alt="P C A 9685 I 2 C Interface" resize="148x120" declaredimensions=true >}}
+        <p>PCA9686 I<sup>2</sup>C Interface</p>
+      </a>
+    </li>
+  </ul>
+  <ol style="visibility: hidden" aria-hidden="true">
+    <li><a href="#c1_slide0">Raspberry Pi 5</a></li>
+    <li><a href="#c1_slide1">Raspberry Pi 4</a></li>
+    <li><a href="#c1_slide2">NVIDIA Jetson Orin Nano</a></li>
+    <li><a href="#c1_slide3">Intel UP4000</a></li>
+    <li><a href="#c1_slide4">Mediatek Genio 500 Pumpkin</a></li>
+    <li><a href="#c1_slide5">Espressif ESP32</a></li>
+    <li><a href="#c1_slide6">Texas Instruments TDA4VM</a></li>
+    <li><a href="#c1_slide7">NVIDIA Jetson Nano</a></li>
+    <li><a href="#c1_slide8">NVIDIA Jetson AGX Orin</a></li>
+    <li><a href="#c1_slide9">NVIDIA Jetson Xavier NX</a></li>
+    <li><a href="#c1_slide10">Raspberry Pi 3</a></li>
+    <li><a href="#c1_slide11">Raspberry Pi Zero 2 W</a></li>
+    <li><a href="#c1_slide12">BeagleBone AI-64</a></li>
+    <li><a href="#c1_slide13">Numato GPIO Modules</a></li>
+    <li><a href="#c1_slide14">PCA9686 I<sup>2</sup>C Interface</a></li>
+  </ol>
+  <div class="prev" style="display: block">‹</div>
+  <div class="next" style="display: block">›</div>
+</div>
+<br>
+
+Then pull together whichever Viam resources you like by configuring your smart machine.
+Finally, program your smart machine with an SDK in your preferred coding language.
```

Comment:
Let's combine this and then just have the cards do the rest
```suggestion
Then pull together the Viam resources you need by configuring your smart machine before programming your machine:
```

Reply:
That feels like a lot in a single sentence. Also I was hoping to provide a little more of a step-by-step call to action situation.

I could move the board carousel out of here onto the platform page, and turn the "Start building" section into an ordered list of 3 steps. Each step would have a link, and the cards would be removed entirely.

Reply:
my issue with it is that there's two sentences that feel like they should have links but don't because the cards are immediately after. You can keep the two sentences if you prefer but you do need to get rid of the `whichever` and `you like`, that is a bit too colloquial

Reply:
```suggestion
Then pull together the Viam resources you want to use by configuring your smart machine.
Finally, program your smart machine with an SDK in your preferred coding language.
```